modules/light-clients/07-tendermint/types/store.go

```diff
@@ -59,29 +58,23 @@ func SetConsensusState(clientStore sdk.KVStore, cdc codec.BinaryCodec, consensus
 
 // GetConsensusState retrieves the consensus state from the client prefixed
 // store. An error is returned if the consensus state does not exist.
-func GetConsensusState(store sdk.KVStore, cdc codec.BinaryCodec, height exported.Height) (*ConsensusState, error) {
+func GetConsensusState(store sdk.KVStore, cdc codec.BinaryCodec, height exported.Height) (*ConsensusState, bool) {
 	bz := store.Get(host.ConsensusStateKey(height))
 	if bz == nil {
-		return nil, sdkerrors.Wrapf(
-			clienttypes.ErrConsensusStateNotFound,
-			"consensus state does not exist for height %s", height,
-		)
+		return nil, false
 	}
 
 	consensusStateI, err := clienttypes.UnmarshalConsensusState(cdc, bz)
 	if err != nil {
-		return nil, sdkerrors.Wrapf(clienttypes.ErrInvalidConsensus, "unmarshal error: %v", err)
+		return nil, false
```

Comment:
should we panic?

Reply:
Wondered that too myself, was considering putting it in this PR, but I figured I'd wait to discuss here. 

I think its reasonable enough to panic if unmarshalling or type assertion fails. It *should* be unreachable code anyway, but in the event it fails to unmarshal for example something has clearly gone wrong..

Happy to panic here if you are. Any thoughts @seantking ?

Reply:
I think panic here makes sense. I don't see a reason to err gracefully in this situation :thinking: